src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -134,7 +134,7 @@ public function load($data, $type = null): RouteCollection
                             'collection' => $operation['collection'],
                             'operationId' => $operationId,
                         ],
-                    ] + ($operation['defaults'] ?? []),
+                    ] + ($operation['defaults'] ?? []) + ($resourceMetadata->getStateless() ? ['_stateless' => true] : []),
```

Comment:
This doesn't seem correct to me. We wouldn't want to have special cases like this.

Reply:
Yes I get it, but how do you think we should handle it ?

Reply:
As discussed together, it should be put in resource metadata attributes instead